src/components/CharacterButton.vue

```diff
@@ -16,6 +25,28 @@
       transition-hide="none"
     >
       <q-list>
+        <q-item
+          v-if="selectedStyleInfo == undefined"
+          class="row no-wrap items-center"
+        >
+          <span class="text-warning vertical-middle">{{
+            characterInfos.length === 0
+              ? "選択可能なスタイルがありません"
+              : "有効なスタイルが選択されていません"
+          }}</span>
+        </q-item>
```

Comment:
この実装でもユーザーにはあまり問題ないのですが、コード的に意図がわかりやすいのはこうかなと思いました！
```suggestion
        <q-item
          v-if="selectedStyleInfo == undefined"
          class="row no-wrap items-center"
        >
          <span class="text-warning vertical-middle">有効なスタイルが選択されていません</span>
        </q-item>
        <q-item
          v-if="characterInfos.length === 0"
          class="row no-wrap items-center"
        >
          <span class="text-warning vertical-middle">選択可能なスタイルがありません</span>
        </q-item>
```